### PR TITLE
+ login, register

### PR DESCRIPTION
**Pull request by giapnguyenitf**
https://dev.framgia.com/redmine/issues/157461

### CODE REVIEW

**Comment by giapnguyenitf** [comment]
ready

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
+@extends('auth.master')
+
+
+@section('content')
```
Comment (trangkarol) [inline resources/views/auth/login.blade.php]: Đừng chừa dòng trắng ở đây e

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                                    {{ Form::email('email', null, ['class' => 'input']) }}
+                                    @if ($errors->has('email'))
+                                        <span class="help-block">
+                                            <strong>{{ $errors->first('email') }}</strong>
+                                        </span>
+                                    @endif
+                                   
+                                </div>
+                                <div class="form-group{{ $errors->has('password') ? ' has-error' : '' }}">
+                                    {{ Form::label('password', trans('label.password')) }}
```
Comment (trangkarol) [inline resources/views/auth/login.blade.php]: I18n: `password`

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="clearfix"></div>
+                    </div>
+                    <div id="sectionB" class="tab-pane fade">
+                        <div class="innter-form">
+                            {{ Form::open(['route' => 'register', 'method' => 'POST', 'class' => 'sa-innate-form']) }}
+                                <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                                    {{ Form::label('name', trans('label.name')) }}
```
Comment (trangkarol) [inline resources/views/auth/login.blade.php]: I18n: `name`

File: resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+                                    {{ Form::text('name', old('name')) }}
+                                    @if ($errors->has('name'))
+                                        <span class="help-block">
+                                            <strong>{{ $errors->first('name') }}</strong>
+                                        </span>
+                                    @endif
+                                </div>
+                                <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                                    {{ Form::label('email', trans('label.email_address')) }}
```
Comment (trangkarol) [inline resources/views/auth/login.blade.php]: I18n: `email`

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                                    {{ Form::email('email', old('email'), ['class' => 'input']) }}
+                                    @if ($errors->has('email'))
+                                        <span class="help-block">
+                                            <strong>{{ $errors->first('email') }}</strong>
+                                        </span>
+                                    @endif
+                                </div>
+                                <div class="form-group{{ $errors->has('password') ? ' has-error' : '' }}">
+                                    {{ Form::label('password', trans('label.password')) }}
```
Comment (trangkarol) [inline resources/views/auth/login.blade.php]: I18n: `password`

File: resources/views/auth/login.blade.php [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+                                    {{ Form::password('password', null, ['class' => 'input']) }}
+                                    @if ($errors->has('password'))
+                                        <span class="help-block">
+                                            <strong>{{ $errors->first('password') }}</strong>
+                                        </span>
+                                    @endif
+                                </div>
+                                <div class="form-group">
+                                    {{ Form::label('password_confirmation', trans('label.confirm_password')) }}
```
Comment (trangkarol) [inline resources/views/auth/login.blade.php]: I18n: `password_confirmation`

File: resources/views/auth/passwords/email.blade.php [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+                <div class="panel-body">
+                    @if (session('status'))
+                        <div class="alert alert-success">
+                            {{ session('status') }}
+                        </div>
+                    @endif
+
+                    <form class="form-horizontal" method="POST" action="{{ route('password.email') }}">
+                        {{ csrf_field() }}
```
Comment (trangkarol) [inline resources/views/auth/passwords/email.blade.php]: html collective

File: resources/views/auth/passwords/email.blade.php [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <div class="col-md-6 col-md-offset-4">
+                                <button type="submit" class="btn btn-primary">
+                                    Send Password Reset Link
```
Comment (trangkarol) [inline resources/views/auth/passwords/email.blade.php]: I18n: `Send Password Reset Link`

File: resources/views/auth/passwords/reset.blade.php [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Reset Password</div>
+
+                <div class="panel-body">
+                    <form class="form-horizontal" method="POST" action="{{ route('password.request') }}">
+                        {{ csrf_field() }}
```
Comment (trangkarol) [inline resources/views/auth/passwords/reset.blade.php]: html collective

**Comment by trangkarol** [comment]
commented

**Comment by giapnguyenitf** [comment]
ready

File: resources/views/auth/passwords/email.blade.php [hunk 10]
```diff
@@ -0,0 +1,43 @@
+@extends('auth.master')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Reset Password</div>
```
Comment (trangkarol) [inline resources/views/auth/passwords/email.blade.php]: I18n: `Reset Password`

File: resources/views/auth/passwords/reset.blade.php [hunk 11]
```diff
@@ -0,0 +1,56 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Reset Password</div>
```
Comment (trangkarol) [inline resources/views/auth/passwords/reset.blade.php]: I18n: `Reset Password`

**Comment by trangkarol** [comment]
commented

**Comment by giapnguyenitf** [comment]
ready

**Comment by trangkarol** [comment]
merged

**Pull request merged**